liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojo.java

```diff
@@ -120,11 +120,16 @@ private File getThinArchiveTarget(File archiveSrc) {
         String appsDirName = getAppsDirectory();
         File archiveTarget = null;
         File appsDir = null;
+        String rootDirectory = serverDirectory;
```

Comment:
I think this variable should by of type `File`. The build is getting the following error:

```
Error:  /home/runner/work/ci.maven/ci.maven/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojo.java:[123,32] incompatible types: java.io.File cannot be converted to java.lang.String
```

Sorry I did not catch that before.